packages/ui/src/proposals/modals/VoteForProposal/VoteForProposalFormModal.stories.tsx

```diff
@@ -25,17 +26,7 @@ interface Props {
 
 const ConnectedVoteForProposalModalForm = ({ id }: { id: string }) => {
   const { proposal } = useProposal(id)
-  return (
-    <VoteForProposalModalForm
-      onNext={() => true}
-      setRationale={() => true}
-      setStatus={() => true}
-      proposalTitle={proposal?.title || ''}
-      proposalType={proposal?.type || ''}
-      proposalRationale={proposal?.rationale || ''}
-      proposalDetails={proposal?.details}
-    />
-  )
+  return <VoteForProposalModalForm context={{}} send={() => undefined} proposal={proposal as ProposalWithDetails} />
```

Comment:
This break the storybook:
![Screenshot from 2022-05-17 12-28-16](https://user-images.githubusercontent.com/6571453/168791146-2f4b77a5-4a81-4b80-b1f0-48c8e5430b0c.png)